README.md

```diff
@@ -1 +1,225 @@
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget App <a name="about-project"></a>
+
+**Budgetary** empowers the end-users to start a new budget tracking account, where he can create categories and operations and see those data linked.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Make use for migrations features from Active Record**
+- **Include Tables for User, Comment, Post, and Like**
+- **Use user authentification and authorizations**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+Note: We are assuming you have pre-installed ruby, WSL, ubuntu and rails!
+
+### Prerequisites
+
+In order to run this project you need:
+
+first at all, install rails:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git@github.com:MasumaJaffery/Budget_App.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Budget_App
+  bundle install
+```
+
+Setting up the Data Base:
+
+```sh
+  rails db:create
+  rails db:migrate
+  rails db:seed
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+<!-- Test are under construction, but we expect to have them completed soon! -->
+
+Before running tests, ensure you've set up the test database:
+
+```sh
+bin/rails db:migrate RAILS_ENV=test
+```
+
+**Note:** Given that the aim of this project is to test the application's UI, to see if it works without side effects. We have therefore had to modify the contents of certain files, which could have the effect of falsifying certain `Unit` tests. 
+We therefore recommend that, rather than running all the tests, you only test the `features` folder and run 
+
+```sh
+bundle exec rspec ./spec
+
+```
+
+### Deployment
+
+The deployment will be available soon !
+https://budgetary.onrender.com/
+
```

Comment:
- [ ] It seems like the deployment process was completed successfully, with the final message indicating "Build successful 🎉." However,

The log you provided primarily contains information about the assets being written to the file system, such as JavaScript and other files. If there is an issue with your deployment, it might not be apparent from this log alone.

To diagnose the issue, you should look for additional error messages or warnings in your deployment logs.

If you have a specific error message or issue, otherwise, please provide more details, and I'll be happy to assist you in troubleshooting the problem further.


if there are no errors anywhere but still can't deploy, please check all the logic as well. Otherwise, **I think It's safe to say that next reviewer would take this account and approve** if there is no issues